trace-recorder/src/main/org/jetbrains/lincheck/trace/recorder/TraceRecorder.kt

```diff
@@ -69,6 +69,7 @@ object TraceRecorder {
         // and 'eventTracker.finishAndDumpTrace()' is called after analysis is disabled
         val desc = ThreadDescriptor.getCurrentThreadDescriptor() ?: return
         desc.removeAsRootDescriptor()
+        Injections.disableGlobalThreadsTracking()
```

Comment:
It probably does not matter here, but in general a good practice to initialize/clean-up resources in reverse order.

So, in `installAndStartTrace` the order is:
```
desc.setAsRootDescriptor()
Injections.enableGlobalThreadsTracking(eventTracker)
```

Then here it should be in reverse order:
```
Injections.disableGlobalThreadsTracking()
desc.removeAsRootDescriptor()
```

Or vice versa, whatever order is more natural here.